integrations/mcp-memgraph/src/mcp_memgraph/server.py

```diff
@@ -10,6 +10,7 @@
 from memgraph_toolbox.tools.trigger import ShowTriggersTool
 from memgraph_toolbox.tools.betweenness_centrality import BetweennessCentralityTool
 from memgraph_toolbox.tools.page_rank import PageRankTool
+# TODO: In the latest version of memgraph-toolbox changed to .utils.logger
```

Comment:
We know the CI passed after these changes. You can fix it now, and we will release the new versions that include the new toolbox at the same time.